controllers/positions.js

```diff
@@ -0,0 +1,77 @@
+const fs = require('fs')
+const Positions = require('../data/positions.json')
+
+const createPosition = (req, res) => {
+  const newPosition = {
+    id: new Date().getTime().toString(),
+    idClient: req.query.idClient,
+    jobDescription: req.query.jobDescription,
+    vacancy: req.query.vacancy,
+    professionalProfiles: req.query.professionalProfiles,
+    status: req.query.status
+    }
+  Positions.push(newPosition)
+  fs.writeFile('./data/positions.json', JSON.stringify(Positions), {}, err => {
+    if (err) {
+      res.status(400).send(err)
+    } else {
+      res.status(201).json(newPosition)
+    }
+  })
+}
+
+const updatePosition = (req, res) => {
+  let updatedPosition
+  const updatedPositions =  Positions.map((position) => {
+    if(position.id === req.query.id) {
+      updatedPosition = {
+        id: req.query.id,
+        idClient: req.query.idClient,
+        jobDescription: req.query.jobDescription || position.jobDescription,
+        vacancy: req.query.vacancy || position.vacancy,
+        professionalProfiles: req.query.professionalProfiles || position.professionalProfiles,
+        status: req.query.status || position.status
+      }
+      return updatedPosition
+    }
+    return position
+  })
+
+  if(updatedPosition === undefined) {
```

Comment:
just write `if(!updatedPosition)` instead of `if(updatedPosition === undefined)`.